composer.json

```diff
@@ -48,5 +57,5 @@
             "dev-master": "2.1-dev"
         }
     },
-    "prefer-stable": true
+    "minimum-stability": "dev"
```

Comment:
this should be in the travis.yml for the versions that actually need it: `before_install: - if ! [ -z "$STABILITY" ]; then composer config minimum-stability ${STABILITY}; fi;`

Reply:
is done now through our dev-kit

Reply:
but i didn't run it here yet.

Reply:
its using the perl thingy however, instead of composer itself.